NICE.Identity.Authorisation.WebAPI/Controllers/UsersController.cs

```diff
@@ -212,7 +211,7 @@ public async Task<IActionResult> DeleteUser(int userId)
             try
             {
                 await _usersService.DeleteUser(userId);
-                return Ok();
+                return Ok(new object());
```

Comment:
any chance of a comment here explaining why please?
i know you told me about this, and I think it just needs a comment